### PR TITLE
Add sheets beta integration

### PR DESCRIPTION
**Pull request by Gerviba**

### CODE REVIEW

File: backend/src/main/resources/config/application.properties [hunk 1]
```diff
@@ -76,13 +76,14 @@ hu.bme.sch.cmsch.component.load.leaderboard=true
...
 hu.bme.sch.cmsch.component.load.messaging=true
-hu.bme.sch.cmsch.component.load.pushnotification=false
 hu.bme.sch.cmsch.component.load.news=true
 hu.bme.sch.cmsch.component.load.profile=true
 hu.bme.sch.cmsch.component.load.proto=true
+hu.bme.sch.cmsch.component.load.pushnotification=false
 hu.bme.sch.cmsch.component.load.qrFight=true
 hu.bme.sch.cmsch.component.load.race=true
 hu.bme.sch.cmsch.component.load.riddle=true
+hu.bme.sch.cmsch.component.load.sheets=true
```
Comment (Isti01) [inline backend/src/main/resources/config/application.properties:86]: Maybe introduce an env var for this?

File: backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsSetupWizard.kt [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+        val token = requestParams.getOrDefault("token", UUID.randomUUID().toString())
+        val name = requestParams.getOrDefault("name", "névtelen integráció")
+        return listOf(
+            permissionCard,
+            DashboardDocsCard(
+                id = 2,
+                title = "Űrlap és Google sheets összekötése",
+                description = "",
+                wide = wide,
+                markdownContent = """
```
Comment (Isti01) [inline backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsSetupWizard.kt:111]: QoL: Add copy button option to card contents

File: backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsUpdaterService.kt [hunk 3]
```diff
@@ -64,7 +64,10 @@ class SheetsUpdaterService(
                 .build()
 
             client.newCall(request).execute().use { response ->
-                if (!response.isSuccessful) throw IOException("Unexpected code $response")
+                if (!response.isSuccessful) {
+                    println(response.body?.string())
```
Comment (Isti01) [inline backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsUpdaterService.kt:68]: Maybe use logger?

File: backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsSetupWizard.kt (hunk 2 above)
Comment (Gerviba) [reply backend/src/main/kotlin/hu/bme/sch/cmsch/component/sheets/SheetsSetupWizard.kt:111]: good idea

**Pull request merged**